index.js

```diff
@@ -1,11 +1,51 @@
 class SortedList {
-  constructor() {}
-  add(item) {}
-  get(pos) {}
-  max() {}
-  min() {}
-  average() {}
-  sum() {}
-};
+  constructor(items, length) {
+    this.items = [];
+    this.length = 0;
+  }
+  add(x) {
+    this.items.push(x);
+    this.length++;
+    this.items.sort((a, b) => {
+      return a - b;
+    });
+  }
+  get(i) {
+    i = i - 1;
+    if (this.items[i] != null) {
```

Comment:
ésta condición se cumple siempre, no hace lo que quieres. Prueba con esto:
if (this.items[i] !== undefined) {